### DISCUSSION TITLE
POH jewellery box shift click option

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#15346] [Ideas]
can you guys add an option to shift click "previous destination" ? atm the j box is recognized as different objects depending on last destination used. so would require manually visiting each teleport and setting shift click to that. figured would be easier/better to add this feature. also depending on if people have last destination or teleport menu as left click in the 'menu entry swapper' plugin maybe you guys can make the vice versa for shift click.

**Comment by geheur** [comment]
there's a swap in "custom menu swaps" on the hub that accomplishes this specifically for the jewellery box.

That said, a more general solution for items that change ids would be neat, although I don't know how that might be accomplished. This affects other objects such as fairy rings as well. see also https://github.com/runelite/runelite/issues/14864 https://github.com/runelite/runelite/discussions/14966

**Reply by ghost** [reply]
yo thanks hadn't seen the other menu plugins.

**Reply by ghost** [reply]
does not work with shift because the runelite shift thru object overrides this plugin hub plugin.
